jetty-documentation/src/main/asciidoc/administration/http2/introduction.adoc

```diff
@@ -42,3 +42,9 @@ The Jetty HTTP/2 implementation consists of the following sub-projects (each pro
 4.  `http2-client`: Provides the implementation of HTTP/2 client with a low level HTTP/2 API, dealing with HTTP/2 streams, frames, etc.
 5.  `http2-http-client-transport`: Provides the implementation of the HTTP/2 transport for `HttpClient` (see xref:http-client[]).
 Applications can use the higher level API provided by `HttpClient` to send HTTP requests and receive HTTP responses, and the HTTP/2 transport will take care of converting them in HTTP/2 format (see also https://webtide.com/http2-support-for-httpclient/[this blog entry]).
+
+____
+[CAUTION]
+If you plan to run your Jetty HTTP/2 application from a fat (uber) jar file, your build tool will likely need to merge some jetty configuration files.
```

Comment:
Can we make this:

fat (uber) jar or shaded jar

will need to merge and META-INF/services and META-INF/versions directories.

Reply:
I'm not so sure that we need to say anything here.   This warning applies to many aspects of Jetty.